MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
+
+include requirements.txt
```

Comment:
Not needed if we remove [L6-10](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L6) and corresponding  imports in `setup.py` ([line 1](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L1) and [line 3](https://github.com/clementchadebec/benchmark_VAE/blob/main/setup.py?plain=1#L3))

Reply:
Yes you are right. I think commenting those lines is the right approach so we can easily revert if needed.

I will indeed keep the dependencies inside `setup.py` for now since there are not that many. So we can either keep , remove or comment the `include requirements.txt`. As you think best.